src/requirements.txt

```diff
@@ -32,4 +32,4 @@ requests==2.25.1
 # git+https://github.com/hubmapconsortium/commons.git@${COMMONS_BRANCH}#egg=hubmap-commons
 hubmap-commons==2.0.14
```

Comment:
Can we upgrade to the latest 2.0.15 release of the hubmap-commons?

Reply:
Yes. I will test.  Isn't this actually independent of the S3 work?  What would initiate this change, if it didn't coincide with a PR I was making?

Reply:
The hubmap-commons update is independent, has nothing to do with the PR. I just noticed that and thought if we can wrap it into your PR rather than doing it separately. Sorry that I didn't make it clear and I can do a separate commit to the main branch (the new release of commons just had a bug fix to the membership lookup, won't cause any side effects) if you don't want to introduce unrelated changes.

Reply:
Nope, not a problem.  Just wanted to confirm that, if I'm specifying version numbers in requirements.txt, the process for maintaining & upgrading wasn't just waiting for other work, like this PR.